handlers/file_upload.js

```diff
@@ -98,15 +101,40 @@ module.exports = {
 
             async.series(
                {
+                  // Scan for malware
+                  clamav: (next) => {
+                     if (!process.env.CLAMAV_ENABLED == "true") {
```

Comment:
random question:  do these environment variables come in as strings or boolean?

Reply:
They're strings.